### ISSUE TITLE
Additional information: Install instructions

### ISSUE THREAD

**Issue by NullChain0** [seyahdoo/baldurs-gate-3-auto-join-conversation#1]
I have seen on several reddit forums and other places that people have not entirely understood on how to get this working while relatively simple some people might get confused.

Hence a tiny bit of documentation for this on the GIT page is helpful for everyone!

For windows users please do the following:
Download both the [baldurs-gate-3-auto-join-conversations.exe](https://github.com/seyahdoo/baldurs-gate-3-auto-join-conversation/releases/download/v1.0.2/baldurs-gate-3-auto-join-conversations.exe) As well as the [the source code zip](https://github.com/seyahdoo/baldurs-gate-3-auto-join-conversation/archive/refs/tags/v1.0.2.zip)

Go ahead and unzip the source code into a separate folder and open up both pictures (this is important!)
Make sure the pictures are not visible on the desktop unless you want to test if it works.
Launch the exe file "baldurs-gate-3-auto-join-conversations.exe"

This should now work if done correctly 
https://i.imgur.com/RzARMS0.png much like this; if it works it should come up with that message and move your cursor to the little icon.
(Make sure its tucked away neatly and minimized!!! (The pictures whilst having the exe running!)

Now launch baldurs gate 3, and this should make it work! If it still has issues please report it and open as issue. The original dev or I will try and help out 👍 

Thank you for helping everyone out here with this tiny but insanely useful functionality Seyahdoo!
I hope this helps everyone trying to run it.

TLDR;

1. Download EXE and Source zip
2. Extract source zip into seperate folder and open both png files
3. Minimize both png files to the background but leave them opened
4. Open the exe file you downloaded
5. Wait until it says Trying to locate listen in button
6. (Optional) Bring one of the pictures to front and if your mouse moves to the picture and the CMD says clicking on ... it works!
7. Launch baldurs gate 3 and enjoy!

**Comment by seyahdoo** [comment]
Thanks for your contribution. It brings me joy to see this tool being useful for the community. 

With the last two release I think this is not necessary anymore, you should be able to just download the zip and open the exe inside.

https://github.com/seyahdoo/baldurs-gate-3-auto-join-conversation/releases/tag/v1.1.1

**Comment by NullChain0** [comment]
Fantastic @seyahdoo ! Appreciate the hard work :-) ! 👍 
A little side issue I might open something up for. When given PRIVATE dialogue the game still wants to click the join conversation button.. I don't know if this is solvable especially when in camp and you want some private dialogue to happen there. 

- Ontop of that maybe a little hotkey for disabling-enabling the program? for in certain situations you might want to join (in range) when you're out of range so you can waddle to it with said character then join up (pressing key is easier than hovering your mouse all the way on to the join button, yep I am lazy like that 👍 and just having the functionality to disable it temporarily in certain situations is a godsent)

**Comment by Chigodfather** [comment]
Do all players need to install this or just the host?

**Comment by seyahdoo** [comment]
> Do all players need to install this or just the host?

any player who would like to auto join the conversations should run this

**Comment by Shootnloot** [comment]
> > Do all players need to install this or just the host?
> 
> any player who would like to auto join the conversations should run this

When playing in split screen coop with controllers the icon is not an ear but instead a speech bubble. Is there anyway you could release a version that would also pick up that icon as well? Also not sure if it would work with two people using controllers or if it would require a keyboard and mouse to be used for the cursor selection of the icon once its sensed. Thanks again for taking the time to make this program :) 

I have attached a picture for reference
![Speech Bubble Icon](https://github.com/seyahdoo/baldurs-gate-3-auto-join-conversation/assets/143853199/33be07dc-c3eb-4f70-bd57-b6f6d4c000a7)

**Comment by IllumiC33** [comment]
When I run the mod, my cursor keeps jumping and giving movement inputs and this also happens in dialogue which makes it very hard to choice options.

**Comment by seyahdoo** [comment]
Hello @IllumiC33 ^^ You can try to reduce the confidence on the settings file. Im gonna reduce the default confidence a bit on the next release
